web/src/components/ToolbarItem.tsx

```diff
@@ -1,18 +1,25 @@
 import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
 import styled from 'styled-components'
 
 import dimensions from '../constants/dimensions'
+import { spacesToDashes } from '../utils/stringUtils'
 import CleanAnchor from './CleanAnchor'
 import StyledSmallViewTip from './StyledSmallViewTip'
 import Button from './base/Button'
 import Icon from './base/Icon'
+import Tooltip from './base/Tooltip'
 
-const StyledToolbarItem = styled(CleanAnchor)`
+const StyledToolbarItem = styled(CleanAnchor)<{ disabled?: boolean }>`
   display: inline-block;
   padding: 8px;
-  cursor: pointer;
+
+  && {
+    cursor: ${props => (props.disabled ? 'default' : 'pointer')};
+  }
```

Comment:
Okay, so I was thinking why we need this special selector and it is because the item is displayed as an `a` tag, even though it is disabled. I think we should not do that and instead render it as a regular `div` for accessibility and so on. See below.